pkg/lang/ir/editor.go

```diff
@@ -85,7 +86,7 @@ func (g Graph) generateJupyterCommand(workingDir string) []string {
 			"--NotebookApp.token", "''")
 	}
 	if g.JupyterConfig.Port != 0 {
```

Comment:
The `if` statement is not necessary.

Reply:
I think so, please help fix this if you are interested.

Reply:
https://github.com/tensorchord/envd/pull/773